src/fontra/client/web-components/add-remove-buttons.js

```diff
@@ -0,0 +1,47 @@
+import { html, css, LitElement } from "https://cdn.jsdelivr.net/npm/lit@2.6.1/+esm";
+
+export class addRemoveButtons extends LitElement {
```

Comment:
`addRemoveButtons` is a class name and should be capitalized: `AddRemoveButtons`